examples/convert_checkpoint/simple_multi_converter.py

```
@@ -0,0 +1,122 @@
+        self.label = label
+        self.targets = targets or ["re:.*"]  # default to all layers
+
+    def process(self, tensors: dict[str, torch.Tensor]):
+        """Print info about targeted tensors."""
+        matched = list(
+            match_quantizable_tensors(tensors, [], self.targets, allow_nonquantizable=True)
+        )
+        print(f"\n[{self.label}] Processing {len(matched)} tensors")
+        if matched:
+            # Show first few as examples
+            for module_name, tensor_name in matched[:3]:
```

Ruff (0.15.12)

[warning] Loop control variable `module_name` not used within loop body

Rename unused `module_name` to `_module_name`

(B007)